lib/Runtime/Types/DictionaryTypeHandler.cpp

```diff
@@ -2460,7 +2538,8 @@ namespace Js
     }
 
     template <typename T>
-    void DictionaryTypeHandlerBase<T>::InvalidateFixedField(DynamicObject* instance, PropertyId propertyId, DictionaryPropertyDescriptor<T>* descriptor)
+    template <typename TPropertyKey>
```

Comment:
T is still there :) one line above TPropertyKey

---
In reply to: [79702610](https://github.com/Microsoft/ChakraCore/pull/1542#discussion_r79702610) [](ancestors = 79702610)